runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -1307,9 +1311,9 @@ TR::Register *J9::X86::TreeEvaluator::multianewArrayEvaluator(TR::Node *node, TR
       generateRegImmInstruction(ANDRegImm4(), node, temp1Reg, -elementSizeAligned, cg);
       }
 
-   // temp2Reg = firstDimLenReg * 16 (discontiguousArrayHeaderSizeInBytes)
-   TR_ASSERT_FATAL(zeroArraySize >= 0 && zeroArraySize <= 127, "discontiguousArrayHeaderSizeInBytes cannot be > 127 for IMulRegRegImms instruction");
-   generateRegRegImmInstruction(IMULRegRegImm4(), node, temp2Reg, firstDimLenReg, zeroArraySize, cg);
+   // temp2Reg = firstDimLenReg * alignedDiscontiguousArrayHeaderSizeInBytes
```

Comment:
Delete this commented line.

Reply:
Revised in commit e98874bf5b1be4188e76f129a472c4a0434cd525